modules/era/lua/era_pet_reward.lua

```diff
@@ -0,0 +1,141 @@
+-----------------------------------
+-- Era pet reward ability
+-----------------------------------
+require('modules/module_utils')
+-----------------------------------
+local m = Module:new('era_pet_reward')
+
+m:addOverride('xi.actions.abilities.reward.onUseAbility', function(player, target, ability, action)
```

Comment:
This was moved to the job_utils folder. Lets override the function in there instead to keep things consistent and move this into the ASB/job_utils folder.